src/services/completions.ts

```diff
@@ -890,6 +928,9 @@ namespace ts.Completions {
         if (isRightOfDot) {
             getTypeScriptMemberSymbols();
         }
+        else if (isRightOfQuestionDot) {

```

Comment:
Probably `isRightOfDot || isRightOfQuestionDot` instead

Reply:
👍 I was previously doing something additional in this branch, but just moved that logic elsewhere.